decidim-elections/app/views/decidim/votings/polling_officer_zone/in_person_votes/_in_person_form.html.erb

```diff
@@ -19,9 +19,9 @@
       <%= f.label t(".date_of_birth") %>
 
       <div class="flex gap-2">
-        <%= f.text_field :day, placeholder: t(".day_placeholder"), label: t(".day"), "data-autojump" => true, "data-max-length" => 2, "data-jump-next" => "#in_person_month" %>
-        <%= f.text_field :month, placeholder: t(".month_placeholder"), label: t(".month"), "data-autojump" => true, "data-max-length" => 2, "data-jump-prev" => "#in_person_day", "data-jump-next" => "#in_person_year" %>
-        <%= f.text_field :year, placeholder: t(".year_placeholder"), label: t(".year"), "data-autojump" => true, "data-max-length" => 4, "data-jump-prev" => "#in_person_month" %>
+        <%= f.text_field :day, placeholder: t(".day_placeholder"), label: t(".day"), data: { autojump: true, max: { length: 2}, jump: { next: "#in_person_month" } } %>
+        <%= f.text_field :month, :placeholder => t(".month_placeholder"), :label => t(".month"), "data-autojump" => true, "data-max-length" => 2, "data-jump-prev" => "#in_person_day", "data-jump-next" => "#in_person_year" %>
+        <%= f.text_field :year, :placeholder => t(".year_placeholder"), :label => t(".year"), "data-autojump" => true, "data-max-length" => 4, "data-jump-prev" => "#in_person_month" %>
```

Comment:
We need to do it here as well lines 23 & 24, but we could di it in a separate PR, as my feeling is that we have a lot of places.

Reply:
I think we could ignore it as they're actually from `decidim-elections` and that was extracted to the `decidim-bulletin-board` repository